app/components/relatedRecords/RelatedPlaces.tsx

```diff
@@ -139,7 +158,11 @@ const RelatedPlaces = () => {
             return (
               <div key={`related-place-${relatedPlace.uuid}`}>
                 <button
-                  className={`text-black/75 hover:text-black text-left md:py-1 ${activePlace === relatedPlace ? "underline font-bold" : ""}`}
+                  className={`text-black/75 text-left md:py-1 ${
```

Comment:
This looks good. I would like to see the tooltips show when hovering the names here on the left.